docs/installing-solidity.rst

```diff
@@ -232,7 +328,7 @@ The following C++ compilers and their minimum versions can build the Solidity co
 - `Clang <https://clang.llvm.org/>`_, version 3.4+
 - `MSVC <https://visualstudio.microsoft.com/vs/>`_, version 2019+
 
-Prerequisites - macOS
+Prerequisites - MacOS
```

Comment:
Actually `macOS` is the official styling as of late.

Reply:
Good to know. I was never sure if I should use `macOS`, `Mac OS`, `MacOS X` or something in between.